openedx/core/djangoapps/user_api/accounts/tests/retirement_helpers.py

```diff
@@ -158,6 +159,28 @@ def _get_dead_end_states(self):
         return [state for state in RetirementState.objects.filter(is_dead_end_state=True)]
 
 
+class RetirementTestUserMixin(TestCase):
```

Comment:
Mixins shouldn't (in my opinion) have a base class other than `object`

```suggestion
class RetirementTestUserMixin(object):
```

Reply:
Done